plutus-playground-lib/src/Playground/Schema.hs

```diff
@@ -40,7 +42,8 @@ instance EndpointToSchema Empty where
 
 instance (ToSchema params, KnownSymbol label, EndpointToSchema (R bs)) =>
          EndpointToSchema (R (label :-> (EndpointValue params, ActiveEndpoints) : bs)) where
-    endpointsToSchemas = FunctionSchema {..} : endpointsToSchemas @(R bs)
+    endpointsToSchemas =
+        FunctionSchema {endpointName, arguments} : endpointsToSchemas @(R bs)
```

Comment:
why not `RecordWildCards` any more?

Reply:
A few people in the team have been complaining about it, because it hides information in the same way that `import Types (Foo(..))` does. I'm kinda sold.